compiler-rt/lib/scudo/standalone/common.h

```diff
@@ -112,6 +112,21 @@ template <typename T> inline void shuffle(T *A, u32 N, u32 *RandState) {
   *RandState = State;
 }
 
+inline void computePercentage(uptr Numerator, uptr Denominator, uptr *Integral,
```

Comment:
I was thinking if we want a `Pair` in Scudo but I didn't get a good use case to add it. This form works for me too. Will leave it to you to make the decision.

Reply:
Yeah, I was hoping I could use std::pair, but I think that would be a bad idea. And adding our own definition feels like a lot of work for a single case that's not necessary.

Reply:
Yeah, I agree with this